crowdsec-docs/unversioned/console/am-i-under-attack.md

```diff
@@ -0,0 +1,49 @@
+# Am I Under Attack
+
+## Introduction
+
+The feature Am I Under Attack is the pinnacle of what CrowdSec offers in terms of real-time infrastructure monitoring.
```

Comment:
I dont generally like this wording "pinnacle", has mkt approved this? if not can I refactor?

Reply:
No that's from me, you can rephrase it if you think it's not really suitable